src/GafferUIModule/AnimationGadgetBinding.cpp

```diff
@@ -70,14 +75,35 @@ void setEditablePlugs( AnimationGadget &animationCurveGadget, object plugs )
 	animationCurveGadget.setEditablePlugs( p );
 }
 
+struct BinarySlotCaller
+{
+
+	boost::signals::detail::unusable operator()( boost::python::object slot, Gaffer::PlugPtr p, bool b )
+	{
+		try
+		{
+			slot( p, b );
+		}
+		catch( const error_already_set &e )
+		{
+			PyErr_PrintEx( 0 ); // clears the error status
```

Comment:
Just spotted this - it should be `translatePythonException()`, but if we use StandardSets like I suggested, I don't think we'll need this signal anyway...